extensions/configuration-editing/src/extension.ts

```diff
@@ -61,11 +64,25 @@ function registerSettingsCompletions(): vscode.Disposable {
 	});
 }
 
-function newSimpleCompletionItem(text: string, range: vscode.Range, description?: string): vscode.CompletionItem {
+function registerExtensionsCompletions(): vscode.Disposable {
+	return vscode.languages.registerCompletionItemProvider({ pattern: '**/extensions.json' }, {
+		provideCompletionItems(document, position, token) {
+			const location = getLocation(document.getText(), document.offsetAt(position));
+			const range = document.getWordRangeAtPosition(position) || new vscode.Range(position, position);
+			if (location.path[0] === 'recommendations') {
+				return vscode.extensions.all
+					.filter(e => !e.id.startsWith('vscode.'))
```

Comment:
- You also need to exclude following extension which is packaged along with VS Code - `Microsoft.vscode-markdown`. This was getting shown in the intelli-sense.

- Intelli-sense should not show already entered extensions.